### PR TITLE
Fix punc-caps issues of cli Tagline

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

File: core/commands/config.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ type ConfigField struct {
 
 var ConfigCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "get and set IPFS config values",
+		Tagline: "get and set IPFS config values.",
```
Comment (RichardLitt) [inline core/commands/config.go]: Caps on get?

**Comment by RichardLitt** [comment]
One comment, otherwise LGTM.

File: core/commands/config.go (hunk 1 above)
Comment (rht) [inline core/commands/config.go]: Somehow missed that.
Used regex /Tagline:.*"[[:lower:]]/ to fix them once for all.

**Comment by RichardLitt** [comment]
LGTM.

**Comment by whyrusleeping** [comment]
@rht please let either @jbenet or myself merge these.

**Comment by jbenet** [comment]
^ 

and LGTM

**Pull request merged**